guides/install-console-application.md

```diff
@@ -2,22 +2,46 @@
 category: Using Form Builder
 expires: 2020-03-31
 ---
- 
+
+# Installing Git
+
+Git is a dependency of the Editor. Below I show two ways to install Git.
+
+## Option 1: XCode
+
+Apple ships a binary package of Git with XCode.
+
+Open your terminal app (Command + Space then type Terminal + Enter)
+Type the following:
+
+    ```
+      xcode-select --install
+    ```
+
+Select "Yes" to install Git and wait until it finishes.
+
+## Option 2: Binary installer
+
+Tim Harper provides an installer for Git.
+You can download [here](https://sourceforge.net/projects/git-osx-installer/).
+After download, open the *.dmg file and double click the *.pkg file
+and follow the instructions.
+
```

Comment:
Do we tell the MoJ staff to use the git client in Self Serve?

Reply:
Not that I am aware of. Should I include here an indication fo git clients? Or any particular git client we should recommend?

Reply:
There is only Github desktop in the self service

Reply:
Done!